src/app/shared/menu/dropdown.css

```diff
@@ -43,8 +44,16 @@
 .open .dropdown-content {
   padding: 20px;
   max-height: initial;
-  transition: max-height;
-  transition: 0.5s;
+  overflow-y: auto;
+}
+.open .dropdown-content.rollout {
+  height: auto;
+  max-height: 500px;
+  transition: max-height 0.2s;
```

Comment:
fixes https://github.com/PRX/metrics.prx.org/issues/145 (those were meant to be `transition-property` and `trasition-duration` oops, and 0.5s was way too slow)

Reply:
I'm in no hurry!